src/app/shared/shapers/discrete-bar-chart.ts

```diff
@@ -0,0 +1,32 @@
+import * as R from 'ramda';
+
+export const shape = ({ key, values }) => {
+  return [{ key, values }];
+};
```

Comment:
the only reason I wrote it the way I did was because I wasn't sure if I'd need to do more shaping long term or not. assuming we get to the point that this is no longer WIP and is ready to commit, I'll convert to that